ldms/src/sampler/zfs_common/Makefile.am

```diff
@@ -0,0 +1,14 @@
+noinst_LTLIBRARIES =
+dist_man1_MANS =
+
+AUTOMAKE_OPTIONS = subdir-objects
+
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@ \
+	      -I/usr/include/libspl \
+	      -I/usr/include/libzfs
+
+noinst_LTLIBRARIES += libzfs_common_helper.la
+libzfs_common_helper_la_SOURCES = zfs_common.c zfs_common.h
+libzfs_common_helper_la_LIBADD = $(CORE_LIBADD)
```

Comment:
Delete this line, and the following lines.

Reply:
done